build.gradle

```diff
@@ -284,3 +284,47 @@ bootstrap.dependsOn installTestGems
 runIntegrationTests.shouldRunAfter tasks.getByPath(":logstash-core:test")
 check.dependsOn runIntegrationTests
 
+String elasticsearchSnapshotURL = "https://snapshots.elastic.co/downloads/elasticsearch/elasticsearch-${version}-SNAPSHOT.tar.gz"
+String elasticsearchDownloadLocation = "${projectDir}/build/elasticsearch-${version}-SNAPSHOT.tar.gz"
+
+task downloadEs(type: Download) {
```

Comment:
I've left these ES tasks unmasked, since I anticipate we'll use them for @original-brownbear 's work for distributed plugin execution